examples/react-native/storybook/stories/Checkbox.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { useState } from 'react';
+import { storiesOf } from '@storybook/react-native';
+import { Checkbox } from '@aws-amplify/ui-react-native/dist/primitives';
+
+const StatefulCheckbox = ({ ...props }: any) => {
+  const [value, setValue] = useState(props.value);
+  const onChange = (nextValue: string) => {
+    setValue(nextValue);
+  };
+
+  return <Checkbox {...props} value={value} onChange={onChange} size={20} />;
```

Comment:
Do we think the `Checkbox` should have a `size` prop that matches the [RWA](https://ui.docs.amplify.aws/react/components/checkboxfield#sizes)?

Reply:
Currently the size prop lives on the IconButton but good call, will address this with the Checkbox refactor once we have Icon.